tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,78 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def adding_label(instance):
+    """
+    Method add multiple labels in the node
+    Args:
+        instance: cluster instance
+        node: cluster node
+    Returns:
+        boolean
+    """
+
+    node = instance.cluster.get_nodes()[0].hostname
+    out, _ = instance.installer.exec_command(
+        cmd="cephadm shell ceph orch host add "
+        + node
+        + " "
+        + node
+        + " node-exporter crash osd installer grafana prometheus alertmanager",
+        sudo=True,
+    )
+    if "Added host" in out:
+        log.info("Successfully added multiple roles in same admin node ")
+        return 0
+    log.info("Adding multiple roles in same admin node fail")
+    return 1
+
+
+def run(ceph_cluster, **kw):
+    """Verify removal of _admin node acting as a bootstrap node
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83574670
+    Test Case Flow:
+    1. Deploy a cluster and check for admin node
+    2. Try to remove the _admin node from bootstrap node
+    """
+
+    try:
```

Comment:
This try and catch block makes debugging harder as it's raising general exception. I would suggest to handle unexcpected results at appropriate steps.

Reply:
Thanks @vamahaja
Updated
Remove try and except block